src/views/Onboarding/OnboardingPassword.vue

```diff
@@ -53,6 +53,10 @@ export default {
       if ((this.password.length === this.confirmPassword.length) && (this.password !== this.confirmPassword)) return true
       return false
     },
+    passwordLengthError () {
+      if (this.password && this.password.length < 8) return true
```

Comment:
I'm a proponent of always using braces because if you (or someone else) ever expands the statement it will be required. Missing them could cause nasty and undesired issues in the future.

Also, what do you think about using optional chaining ? 
`if (this.password?.lenght < 8)`

Reply:
Yeah, of course, we can do that.

Reply:
Updated, @kraikov .

Reply:
Amazing @Jennievon 

When you add optional chaining, you no longer need the first check.

So 
`if (this.password && this.password.length < 8)` is the same as `if (this?.password.length < 8)`

Reply:
just a little fix for a typo, `this.password?.length`